### ISSUE TITLE
Too difficult

### ISSUE THREAD

**Issue by DrMeepster** [Barsik008/PossumBot#42]
as seen by other issues, the simple installation of this bot is too difficult for the unintelligent people that would actually want to kill servers with this thing

**Comment by Barsik008** [comment]
@DrMeepster I don't think that i can implement a bugfix for this.

**Comment by HyperFlame1324** [comment]
You should just give us the invite link.

**Comment by DrMeepster** [comment]
![image](https://user-images.githubusercontent.com/19316085/94738860-e3c55b00-0324-11eb-841b-646e9f43ed2a.png)
@HyperFlame1324 that would make Barsik responsible for what you do

**Comment by Barsik008** [comment]
@HyperFlame1324 You should just follow the readme tutorial.

**Comment by P2468** [comment]
**JUST GIVE US THE _BOT_ INVITE LINK**

**Comment by Barsik008** [comment]
@P2468 You are cringe. Smh my head.

**Comment by rizzini** [comment]
> **JUST GIVE US THE _BOT_ INVITE LINK**

The bot gives you the link when you run it, dumbass.

**Comment by Umeor** [comment]
It's actually really easy. It took me two Google searches to follow instructions, if you really wanted to destroy a server, wouldn't you put some effort into it?

**Comment by Mariic33** [comment]
> It's actually really easy. It took me two Google searches to follow instructions, if you really wanted to destroy a server, wouldn't you put some effort into it?

I totally agree with that. I´m not a programmer or something and it just took me some research and I could use the bot. 
Btw. Thank you @Barsik008 for this super cool bot.

**Comment by D3m0n1um** [comment]
it doesnt work, i just tried it.. gave it all rights but i didnt get the ownership of the server

**Comment by dragonDScript** [comment]
> it doesnt work, i just tried it.. gave it all rights but i didnt get the ownership of the server

1. Who the heck thinks that this bot can overcome the entire DISCORD API and the DISCORD'S BACKEND?

2. "It doesn't work" doesn't help

**Comment by Necromonster** [comment]
> > **JUST GIVE US THE _BOT_ INVITE LINK**
> 
> The bot gives you the link when you run it, dumbass.

no it doesnt?

**Comment by garrettmolina2** [comment]
my bot will not run when i try to run the StartBot file. Anyone have a fix for this?

**Comment by DrMeepster** [comment]
> my bot will not run when i try to run the StartBot file. Anyone have a fix for this?

this is the issue mocking people like you, not the place for you to ask for help without giving any information that could help you at all.

**Comment by garrettmolina2** [comment]
> > my bot will not run when i try to run the StartBot file. Anyone have a fix for this?
> 
> this is the issue mocking people like you, not the place for you to ask for help without giving any information that could help you at all.

when I try to run the file in the folder it simply pops up and then just immediately disappears also I tried the node bot.js command and that did not work because it said there was no such file

**Comment by garrettmolina2** [comment]
> > my bot will not run when i try to run the StartBot file. Anyone have a fix for this?
> 
> this is the issue mocking people like you, not the place for you to ask for help without giving any information that could help you at all.

pls help me

**Comment by Surtrgiantfire111** [comment]
PLEASE HELP ive done everything correctly, except where do i enter the ! commands? and the bot doesnt start either, it just briefly opens and then disappears. thank you
